triton-server/src/notification.ts

```diff
@@ -67,9 +69,7 @@ export const sendNotificationEmail = async (
                         -   Readset blocked status count: ${
                             dataset.blocked_status_count
                         }<br/>
-                        -   Dataset latest released update date: ${formatDateAndTime(
-                            dataset.latest_release_update ?? new Date(),
-                        )}<br/>
+                        -   Dataset latest released update date: ${formatDateAndTime()}<br/>
```

Comment:
I think it's better if you use `dataset.latest_release_update` first and then fallback to `formatDateAndTime()` if it's undefined for some reason.

Reply:
`formatDateAndTime` could help with formatting if it accepts `Date`